MAUI/Cards/getting-started.md

```diff
@@ -135,6 +135,8 @@ public partial class MainPage : ContentPage
 {% endhighlight %}
 {% endtabs %}	
 
+N> This property won't work when adding the SfCardView as a child of SfCardLayout.
```

Comment:
N> This property will not work when adding the SfCardView as a child of SfCardLayout.